sbtplugin/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePlugin.scala

```diff
@@ -21,9 +21,15 @@ object ScalaNativePlugin extends AutoPlugin {
     val nativeEmitDependencyGraphPath = settingKey[Option[File]](
       "If non-empty, emit linker graph to the given file path.")
 
-    val nativeLinkAs = settingKey[Map[String, String]](
+    val nativeLibraryLinkage = settingKey[Map[String, String]](
       "Given a native library, provide the linkage kind (static or dynamic). " +
       "If key is not present in the map, dynamic is picked as a default.")
+
+    val nativeLink = inputKey[Unit](
+      "Generates native binary without running it.")
```

Comment:
Sounds good! I guess I'll start with non-attributed file for now and change it later on if that becomes a problem.